src/screens/BookDetail/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useParams, useHistory } from 'react-router-dom';
+import { useQuery } from 'react-query';
+
+import { getBookDetail } from 'services/BooksService';
+import Spinner from 'components/Spinner';
+
+import backButton from './assets/back-arrow.png';
+import styles from './styles.module.scss';
+
+type BookDetailParams = {
+  id: string;
+};
+
+function BookDetail() {
+  const history = useHistory();
+  const { id } = useParams<BookDetailParams>();
+  const { isLoading, data } = useQuery('book-detail', () => getBookDetail(id));
+  return (
+    <div className={styles.container}>
+      <div onClick={() => history.goBack()} className="row middle">
+        <img src={backButton} />
+        <p className={styles.backButton}>Atrás</p>
+      </div>
+      {isLoading && <Spinner containerClassName={styles.loader} />}
+      {data && (
+        <div className={`row ${styles.bookContainer}`}>
+          <img src={data.imageUrl} alt={data.title} className={styles.bookCover} />
```

Comment:
This alt is fine. However sometimes it's necessary to look at the context of the element, some lines  below there's a `<p>` containing the same text.
If a user's is using a screenreader (for accessibility reasons) it will read it 2 times (img alt and paragraph content), so it will result in redundant information. This is beyond the training  so there's no need to change it.

Anyways, if you want it I would recommend using something like `"Book's cover"` or `${data.title}'s cover` (it's is a little redundant anyways tho haha).